composables/useSingleSignIn.ts

```diff
@@ -0,0 +1,41 @@
+export const useSingleSignIn = () => {
+  const singleInstanceServer = useAppConfig().singleInstanceServer
+  const userSettings = useUserSettings()
+  const users = useUsers()
+  const { t } = useI18n()
+
+  const busy = ref(false)
+
+  async function signIn() {
+    if (busy.value)
+      return
+
+    busy.value = true
+
+    await nextTick()
+
+    try {
+      const href = await (globalThis.$fetch as any)(`/api/${publicServer.value}/login`, {
+        method: 'POST',
+        body: {
+          force_login: users.value.length > 0,
+          origin: location.origin,
+          lang: userSettings.value.language,
+        },
+      })
+      busy.value = false
+      location.href = href
+    }
+    catch (err) {
+      console.error(err)
+      busy.value = false
+      await openErrorDialog({
+        title: t('common.error'),
+        messages: [t('error.sign_in_error')],
+        close: t('action.close'),
+      })
+    }
```

Comment:
can we somehow share this code with https://github.com/elk-zone/elk/blob/f3642ed9d54443a9dd73930255d37b938b4e45b8/components/user/UserSignIn.vue#L16-L53?

Reply:
sure: I will check also `entry-*.js` asset size ~500KB (current is about 128KB), maybe we can use dynamic import on nav (maybe due to duplicate code)

Reply:
![imagen](https://user-images.githubusercontent.com/6311119/216764684-e6c208e2-9e4e-4f56-bb62-d401694e75f6.png)